lib/mermaid.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
```

Comment:
Even implementing debouncing, it will flicker while the diagram is updated. Apparently svg render is slow. A solution I can think of is to compute the height once the diagram gets rendered and keep the previous height until the diagram finishes rendering. It will still flicker like appearing disappearing, but the content of the document won't jump over the diagram